src/ws/cockpitwebservice.c

```diff
@@ -1116,10 +1116,7 @@ on_web_socket_open (WebSocketConnection *connection,
   JsonObject *object;
   JsonObject *info;
 
-  if (cockpit_creds_get_rhost (self->creds))
-      g_info ("New connection to session from %s", cockpit_creds_get_rhost (self->creds));
-  else
-      g_info ("New connection to session");
+  g_info ("New connection to session");
```

Comment:
It would be nice to identify sessions somehow, if only with a otherwise meaningless id. That would allow people to interpret the log messages better.  Can we maybe even get the logind session number?

Reply:
That string does not change with this PR in practice, as _get_rhost() is always false-y now. But how about  showing the user name at least? This could be added to all of these:
```
cockpit-ws[1708]: logged in user session
cockpit-ws[1708]: New connection to session
donald cockpit-ws[1708]: connection unexpectedly closed by peer
donald cockpit-ws[1708]: WebSocket from (null) for session closed
```

Reply:
Yes, I was thinking the same.  Let's do that outside of this PR.

Reply:
Done in PR #13249